Neo4j.Driver/Neo4j.Driver/Internal/ConnectionValidator.cs

```diff
@@ -144,4 +144,15 @@ private bool HasBeenIdleForTooLong(IPooledConnection connection)
 
         return false;
     }
+
+
+    public static bool IsTimeoutDetectionEnabled(TimeSpan timeout)
+    {
+        return timeout.TotalMilliseconds >= 0;
+    }
+
+    public static bool IsTimeoutDetectionDisabled(TimeSpan timeout)
+    {
+        return timeout.TotalMilliseconds < 0;
```

Comment:
is a second function needed? And if it is, it would be better to make Disabled => !Enabled so that the logic is only encoded into one place in the code.